File: Pratica 6 - Funcoes 2/java_logic/Exercicio5_v2.java
import java.util.Scanner;

public class Exercicio5_v2 {
    static double converter(double c) {
        return (c * 9/5) + 32;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite a temperatura em Celsius: ");
        double c = sc.nextDouble();
        System.out.println("A temperatura em Fahrenheit é: " + converter(c));

        sc.close();
    }
}

File: Pratica 6 - Funcoes 2/cpp_logic/exercicio5_v2.cpp
#include <iostream>
using namespace std;

float converter(float c) {
    return (c * 9/5) + 32;
}

int main() {
    float c;
    cout << "Digite a temperatura em Celsius: ";
    cin >> c;

    cout << "A temperatura em Fahrenheit é: " << converter(c);

    return 0;
}